users/faq.rst

```diff
@@ -494,3 +494,31 @@ to do this.  The most well known is called daemontools, and can be found in the
 standard package repositories for  almost every modern Linux distribution.
 Other popular tools with similar functionality include S6 and the aforementioned
 runit.
+
+.. _inotify-limits:
+
+How do I increase the inotify limit to get my fs watcher to work?
+-----------------------------------------------------------------
+
+You are probably reading this because you encountered the following error with
+the fs watcher on linux:
+
+    Failed to start filesystem watcher for folder yourLabel (yourID): failed to
+    setup inotify handler. Please increase inotify limits, see
+    https://docs.syncthing.net/users/faq.html#inotify-limits
+
+Linux typically restricts the amount of watches per user (usually 8192). So if
+you have a more directories, you need to adjust that number.
+
+On many Linux distributions you can run the following to fix it: ::
+
+    echo "fs.inotify.max_user_watches=204800" | sudo tee -a /etc/sysctl.conf
+
+On Arch Linux and potentially others it is preferred to write this line into a
+separate file, i.e. you should run ::
```

Comment:
comma after "i.e.", i.e., `... separate file, i.e., you should ...`

(minor :trollface:)

Reply:
I could argue that this is a British vs American thing and I learned most English reading Tolkien, but truth told it just hurts my German-biased language sense (there's never ever a comma after the equivalent "d.h.") - but I digress :)  
Oh and as I am wasting time already:  
```
simon@simon-x220-deb:~/Coding/syncthing-docs$ grep -r "i\.e\.[^,]" | wc -l
105
```
I am not going to waste more time to fix these as well :P

Reply:
You're right, leave this one alone!

https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e